ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -845,6 +850,18 @@ public final class AliasTokens {
         case .warningForeground2:
             return DynamicColor(light: GlobalTokens.sharedColors(.yellow, .shade30),
                                 dark: GlobalTokens.sharedColors(.yellow, .tint30))
+        // Presence
+        case .presenceAway:
+            return DynamicColor(light: GlobalTokens.sharedColors(.marigold, .primary))
```

Comment:
Does presenceAway not have a different dark color?

Reply:
Nope, it uses marigold primary for both light and dark